bots/jabber_mirror_backend.py

```diff
@@ -195,13 +210,16 @@ def nickname_to_jid(self, room, nick):
 
 class ZulipToJabberBot(object):
     def __init__(self, zulip_client):
+        # type: (zulip.Client) -> None
         self.client = zulip_client
         self.jabber = None # type: JabberToZulipBot
 
     def set_jabber_client(self, client):
+        # type: (JabberToZulipBot) -> None
         self.jabber = client
 
     def process_event(self, event):
+        # type: (Dict[str, Any]) -> None
```

Comment:
I am not entirely sure where to look for it. I've already tried the Zulip API to no avail. I can keep trying